utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/JsonSchemaOutputValidatorPrinter.scala

```diff
@@ -0,0 +1,46 @@
+package pl.touk.nussknacker.engine.avro.encode
```

Comment:
Why it's in `avro.encode` package?

Reply:
I will clean this up in on of  the next PRs

Reply:
Ekhm ekhm.. Your decision ;)